src/Controller/NewsController.php

```diff
@@ -32,20 +37,55 @@ public function show(?News $news): Response
 
 
     #[Route('/admin/news', name: 'admin_news')]
-    public function adminNews(NewsRepository $newsRepository): Response
+    public function adminNews(NewsRepository $newsRepository)
     {
         return $this->render('news/admin/index.html.twig', [
             'news' => $newsRepository->findBy(array(), array('id' => 'DESC'), 15)
         ]);
     }
 
     #[Route('/admin/create', name: 'admin_news_created')]
-    public function createNews(): Response
+    public function createNews(Request $request, SluggerInterface $slugger, Security $security, EntityManagerInterface $entityManager, WebhookDiscordService $webhookDiscordService)
     {
+        $this->webhookDiscordService = $webhookDiscordService;
         $news = new News();
-        $newForm = $this->createForm(NewsType::class, $news);
+
+        $form = $this->createForm(NewsType::class, $news);
+        $news->setAuthor($security->getUser());
```

Comment:
Tu es dans un controller qui étend AbstractController. Cette classe justement te donne pleins de raccourcis. En l'occurence $this->getUser() je crois. Fait un ctrl click sur $this->getUser() pour voir un peu comment symfony fait cela.